server/webapp/WEB-INF/rails.new/app/assets/stylesheets/new-theme.scss

```diff
@@ -0,0 +1,1356 @@
+/*
+ * Copyright 2016 ThoughtWorks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+//new theme for old pages in go.cd app
+@import 'https://fonts.googleapis.com/css?family=Open+Sans:300,400,400i,600,700,800';
```

Comment:
No external CDN please — a lot of our users are in a constrained environment and have no access to external networks.

We have open sans available and checked in into the repository. Check the `_fonts.scss` file